README.md

```diff
@@ -73,8 +73,7 @@ data = casparser.read_cas_pdf('/path/to/cas/pdf/file.pdf', 'password')
                             "units": "number",
                             "nav": "number",
                             "balance": "number",
-                            "is_dividend_payout": "boolean",
-                            "is_dividend_reinvestment": "boolean",
+                            "type": "string",
```

Comment:
Presently "type" is returned as TransactionType enum, and not as "string"
I had to use the following in my script to do anything with "type"
`from casparser.enums import TransactionType`

Reply:
Oops you are right. I updated the text while looking at the json output. shall fix this soon.

Reply:
transaction type is string in both data and json now.